dot/network/sync.go

```diff
@@ -43,8 +43,11 @@ func (s *Service) DoBlockRequest(to peer.ID, req *BlockRequestMessage) (*BlockRe
 		return nil, err
 	}
 
-	err = s.host.writeToStream(stream, req)
-	if err != nil {
+	defer func() {
+		_ = stream.Close()
+	}()
```

Comment:
```suggestion
	defer stream.Close()
```

Reply:
this causes a lint error for unchecked err :/